contracts/components/staking/SlashReasons.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: UNLICENSED
+// See Forta Network License: https://github.com/forta-protocol/forta-contracts/blob/master/LICENSE.md
+
+pragma solidity ^0.8.4;
+
+// These are the identifiers for slash reasons. Update this file if new slash reasons are configured, for
+// better reference.
+
+bytes32 constant OPERATIONAL_SLASH = keccak256("OPERATIONAL_SLASH");
```

Comment:
Shouldn't that be an enum ?

Reply:
I went with `bytes32` Ids so governance could add more "types of slashing" without needing to upgrade the contract